cliquet/scripts/init_schema.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import optparse
+import sys
+import textwrap
+
+from pyramid.paster import bootstrap
+
+
+def main():
+    description = """\
+    Initialize storage and cache backend databases. Example:
+    'init_schema deployment.ini'
+    """
+    logging.basicConfig(level=logging.DEBUG)
+    logger = logging.getLogger(__name__)
+
+    usage = "usage: %prog config_uri"
+    parser = optparse.OptionParser(usage=usage,
+                                   description=textwrap.dedent(description))
+    options, args = parser.parse_args(sys.argv[1:])
+
+    if not len(args) >= 1:
```

Comment:
Should we use argparse instead? It could be handy with the subparsers so we can have multiple sub commands, and do something like `cliquet init`, `cliquet migrate` etc.